R/pull_GA_data.R

```diff
@@ -131,7 +154,9 @@ write_df_to_parquet(regionality_metric,
                     sink = 'out/regionality/regionality.parquet')
 
 state_week_vs_year <- compute_week_vs_year(state_traffic_all) %>%
-  rename("365_days" = "365 days", "30_days" = "30 days", "7_days" = "7 days")
+  rename("365_days" = "365 days", "30_days" = "30 days", "7_days" = "7 days") %>% 
+  mutate(weekly_average = `365_days`/52.14,
```

Comment:
is this 52.14 some kind of leap year magic adjustment?

Reply:
Its just 365/7

Reply:
duh yes it is ;)